src/main/webapp/app/teams/teams-status.scss

```
@@ -39,7 +39,7 @@
                 h6 {
                     font-size: calc(6px + 0.8vh);
                 }
-                .team-slogan {
+                .team-title {
```

The title is not vertically centererd on the background image. The margin is correctly applied as follows:
```suggestion
                .team-title-with-slogan {
```